exercises/two-bucket/TwoBucket.cs

```diff
@@ -9,18 +9,18 @@ public enum Bucket
 public class TwoBucketResult
 {
     public int Moves { get; set; }
-    public Bucket GoalBucket { get; set; }
-    public int OtherBucketContents { get; set; }
+    public Bucket Goal_bucket { get; set; }
+    public int Other_bucket { get; set; }
```

Comment:
Please rename this to `OtherBucket`, to conform to standard [.NET naming guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions).